src/models/flight_control/FGActuator.h

```diff
@@ -80,7 +80,7 @@ CLASS DOCUMENTATION
 @code
 <actuator name="name">
   <input> {[-]property} </input>
-  <lag> number </lag>
+  <lag> {property name | value} </lag>
```

Comment:
According to the discussion in issue #291, it is needed to document that the property is not meant to be continuously modified during a simulation.